h2o-docs/src/product/data-science/gam.rst

```diff
@@ -486,7 +490,7 @@ I-splines are used to build the monotone spline functions by restricting the gam
    
```

Comment:
@hannah-tillman For M-spline, there is a special condition that I want the users to know:

If user set spline_orders = 1, they must set num_knots >= 3 instead of >=2.  

Can you put this warning somewhere.